plugins/com.google.cloud.tools.eclipse.bugreport/src/com/google/cloud/tools/eclipse/bugreport/ui/BugReportCommandHandler.java

```diff
@@ -75,4 +78,8 @@ private static String getCloudSdkVersion() {
       return ex.toString();
     }
   }
+
+  private static String getEclipseVersion() {
+    return Platform.getBundle("org.eclipse.platform").getVersion().toString();
```

Comment:
You should handle that the `org.eclipse.platform` bundle is not present (i.e., the bundle returned is `null`).

Although this string isn't the Eclipse version exactly (see _Help > About_, which reports the product name, product version, and build-id), this is simple to compute and is good enough.  Whereas obtaining the _Help > About_ info isn't very easy and requires accessing internal classes.

Reply:
FYI, this returns, e.g., `4.5.2.v20160212-1500`, so I think we get enough info. I'll check null.